pd/migrations/20211216025603_init.sql

```diff
@@ -33,12 +33,23 @@ CREATE INDEX ON notes (position);
 CREATE INDEX ON notes (height);
 
 CREATE TABLE IF NOT EXISTS validators (
-    tm_pubkey bytea NOT NULL PRIMARY KEY,
-    voting_power bigint NOT NULL
+    tm_pubkey bytea NOT NULL PRIMARY KEY
 );
 
 CREATE TABLE IF NOT EXISTS validator_fundingstreams (
     tm_pubkey bytea NOT NULL,
     address varchar NOT NULL,
     rate_bps bigint NOT NULL
 );
+CREATE TABLE IF NOT EXISTS base_rates (
```

Comment:
this is a nit, but it'd be nice to have an extra newline here so that there's one "paragraph" per table + indexes 
```suggestion

CREATE TABLE IF NOT EXISTS base_rates (
```